pyproject.toml

```diff
@@ -25,7 +25,7 @@ license = {text = "Apache 2"}
 dependencies = [
     "duckdb>=1.0.0",
     "datasets>=3.0.0",
-    "openai>=1.50.0",
+    "openai>=1.57.4",
```

Comment:
why did we update this epecifically?

Reply:
oops, my bad! i probably did a `pdm add openai` at some point and pushed the updated pyproject.toml. I will fix that